README.md

```diff
@@ -8,10 +8,18 @@ A brand new default theme for [Hexo].
 
 ### Install
 
+Local git install:
+
 ``` bash
 $ git clone --depth 1 https://github.com/hexojs/hexo-theme-landscape themes/landscape
 ```
 
+npm install:
+
+```
+npm i -P hexo-theme-landscape
```

Comment:
Since version 5 of NPM, you don't need to use the `-P` attribute anymore. This is now the default behavior.